bug-bounty-preview-scopes.md

```diff
@@ -0,0 +1,33 @@
+# .NET Bounty Scope
+
+## Introduction
+
+The [.NET bug bounty](https://www.microsoft.com/en-us/msrc/bounty-dot-net-core) covers current released versions of .NET Core, ASP.NET Core and .NET, as well as the latest beta or release candidate of the upcoming version.
+
+.NET has the concept of [preview features](https://github.com/dotnet/designs/blob/main/accepted/2021/preview-features/preview-features.md) which ship in the current version, or in nightly builds of the upcoming version. Preview features are not enabled by default, but their goal is to be enabled in the next major release. Preview features are now considered in scope for the bug bounty if they are listed in the table below.
+
+Bugs against preview features in the current release must be demonstrated against the latest current release.
+
+Bugs against preview features in nightly builds of the next version must be demonstrated against the build number listed in the table below, or subsequent builds. Preview features in nightly builds may fall out of scope while bugs are addressed or the removal of the feature from the upcoming version.
+
+Any inclusion of a feature in the table below should not be taken as an indication that the feature will ship in any upcoming version.
+
+Documenation of security bugs for preview features are only in scope for the current released version of .NET
+
+Please read the .NET bug bounty [terms and conditions](https://www.microsoft.com/en-us/msrc/bounty-dot-net-core) before submitting your bug report. To be eligible for a bounty you must submit your bug report through the Microsoft Security Response Center (MSRC).
```

Comment:
Cam we be more direct here?

```suggestion
Read the .NET bug bounty [terms and conditions](https://www.microsoft.com/en-us/msrc/bounty-dot-net-core) before submitting your bug report. To be eligible for a bounty, you must submit your bug report through the Microsoft Security Response Center (MSRC).
```

Reply:
Goodness, not often I'm called out for being polite :p

Reply:
LOL - it's Microsoft style guide unless you think this is an inconvenient ask 😄 
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please